src/sentry/models/sentryappwebhookerror.py

```diff
@@ -3,19 +3,17 @@
 from django.db import models
 from django.utils import timezone
 
-from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+from sentry.db.models import EncryptedJsonField, BoundedBigIntegerField, Model
 
 
 class SentryAppWebhookError(Model):
     __core__ = False
 
     date_added = models.DateTimeField(db_index=True, default=timezone.now)
 
-    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="webhook_errors")
+    sentry_app = BoundedBigIntegerField()
 
-    organization = FlexibleForeignKey(
-        "sentry.Organization", related_name="sentry_app_webhook_errors"
-    )
+    organization = BoundedBigIntegerField()
```

Comment:
Learning more things about django migrations here. At the moment, this causes these fields to be renamed from `sentry_app_id` to `sentry_app`, which we definitely don't want.

To fix this, change the columns to `BoundedBigIntegerField(db_column="sentry_app_id")` and `BoundedBigIntegerField(db_column="organization_id")` and regenerate the migration.